File: surf-cloud-standalone/src/main/java/dev/slne/surf/cloud/standalone/netty/TestNettyPacketListener.java
package dev.slne.surf.cloud.standalone.netty;

import dev.slne.surf.cloud.api.meta.SurfNettyPacketHandler;
import org.springframework.stereotype.Component;

@Component
public class TestNettyPacketListener {

  @SurfNettyPacketHandler
  public void handleTestPacket(TestNettyPacket packet) {
    System.out.println("Received Test packet: " + packet);
  }

}

File: surf-cloud-standalone/src/main/kotlin/dev/slne/surf/cloud/standalone/netty/TestNettyPacketListener.kt
package dev.slne.surf.cloud.standalone.netty

import dev.slne.surf.cloud.api.meta.SurfNettyPacketHandler
import org.springframework.stereotype.Component

@Component
class TestNettyPacketListener {
    @SurfNettyPacketHandler
    suspend fun handleTestPacket(packet: TestNettyPacket) {
        println("Received Test packet: $packet")
    }
}
